modules/post/multi/recon/local_exploit_suggester.rb

```diff
@@ -115,35 +225,44 @@ def show_found_exploits
   def run
     if @local_exploits.empty?
       print_error 'No suggestions available.'
+      vprint_line
+      vprint_session_info
+      vprint_status @invalid_modules_table
       return
     end
 
     show_found_exploits
     results = []
-    @local_exploits.each do |m|
+    @local_exploits.each_with_index do |m, index|
```

Comment:
Feels like a lot of this code will be simplified if we separate the data from the presentation concerns, potentially this should just a map:

```suggestion
    results = @local_exploits.map.with_index do |m, index|
```

Then the generation of rex tables would just be a simple `results.partition` + filter/map, or whatever presentational concerns are required. We'd have to track extra metadata in our array of hashes though:

Happy path:
```
{ module: m, checkcode: checkcode }
```

Error path:
```
{ module: m, errors: ['...', '...'] }
```